src/EnergyPlus/SystemAvailabilityManager.cc

```diff
@@ -1852,109 +1785,109 @@ namespace SystemAvailabilityManager {
 
         {
             auto const SELECT_CASE_var(SysAvailType);
-            if (SELECT_CASE_var == SysAvailMgr_Scheduled) { // 'AvailabilityManager:Scheduled'
+            if (SELECT_CASE_var == DataPlant::SystemAvailabilityType::Scheduled) { // 'AvailabilityManager:Scheduled'
```

Comment:
Why no `switch`/`case` here?

Reply:
I missed that one... Fixed in the latest commits. 

Some of the `SELECT_CASE_var` cases were either part of DataHVACGlobals (enum conversion is being done in another work) or were used in [SetupOutputVariable() function](https://github.com/NREL/EnergyPlus/blob/043a3b60a902d6b1bcce8e47e799c07b0a471355/src/EnergyPlus/SystemAvailabilityManager.cc#L4339-L4345) (We don't have a way to use enum class type variables in that function). These were left as `int constexpr`, but the `SELECT_CASE_var` was changed to `switch/case`.